models/prices/blast/prices_blast_tokens.sql

```diff
@@ -16,5 +16,6 @@ FROM
 (
     VALUES
     ('weth-weth','blast','WETH',0x4300000000000000000000000000000000000004,18),
-    ('usdb-usdb','blast','USDB',0x4300000000000000000000000000000000000003,18)
+    ('usdb-usdb','blast','USDB',0x4300000000000000000000000000000000000003,18),
+    ('weth-weth', 'blast', 'bpETH', 0xb772d5c5f4a2eef67dfbc89aa658d2711341b8e5, 18)
```

Comment:
why is this throwing an error? this is for blur pool ETH which is used for bidding on blur and is a wrapped 1:1 to eth, hence why following weth makes sense imo. I added it the same way on ethereum when i added blur v2 on ethereum a while back and its been working all fine there

Reply:
are you using `weth-weth` because `bpETH` doesn't exist on coinpaprika?

that test is used as a safeguard to keep symbols clean, but we've had instances where we override like you're doing here. should be fine to go with a failing test -- on that action, but normal CI should pass. looks like you hit our weird bug we're dealing with on the latest run.

Reply:
yes bpETH doesnt exist on coinpaprika but the distinction is important, same as here for ETH mainnet: https://github.com/duneanalytics/spellbook/blob/29881a9240945a97dd4a53747d3673c9ba92b160/models/prices/ethereum/prices_ethereum_tokens.sql#L1613

Reply:
yes, this is acceptable imo